pom.xml

```diff
@@ -79,6 +79,14 @@
       </dependency>
     </dependencies>
   </dependencyManagement>
+  <!-- Add local repository for private java driver queryBuilder build to avoid using systemPath in dependency-->
```

Comment:
nit: can we include a link to an issue in our repo or the java driver that explains what we are waiting on, and how we can checked if the work has been done. We need the issue to stay open until we revert this change.

Reply:
added tacker issue